src/main/java/com/hotels/beeju/core/BeejuCore.java

```diff
@@ -100,13 +103,44 @@ public BeejuCore(String databaseName, Map<String, String> preConfiguration, Map<
   }
 
   private void configureMisc() {
+    int webUIPort = getWebUIPort();
+
     // override default port as some of our test environments claim it is in use.
-    conf.setIntVar(HiveConf.ConfVars.HIVE_SERVER2_WEBUI_PORT, 0);
+    conf.setIntVar(HiveConf.ConfVars.HIVE_SERVER2_WEBUI_PORT, webUIPort);
     
     conf.setBoolVar(HiveConf.ConfVars.HIVESTATSAUTOGATHER, false);
     
     // Disable to get rid of clean up exception when stopping the Session.
     conf.setBoolVar(HiveConf.ConfVars.HIVE_SERVER2_LOGGING_OPERATION_ENABLED, false);
+
+    // Used to prevent "Not authorized to make the get_current_notificationEventId call" errors
```

Comment:
Nice one for adding the comments explaining why, future code archaeologists will thank you.